### ISSUE TITLE
Social Impact Bonds. (Finance with performance targets)

### ISSUE THREAD

**Issue by BobHarper1** [SocialInvestmentDataLab/spec#21]
Social Impact Bonds (SIBs) are a particular type of finance, unique to the social investment sector, where payment through a contract is made by results, those being specific social outcomes.

#20 got me thinking about how SIBs could be modeled. I would have though they were a type of 'loan' (probably because of the 'bond' name) but according to [this page](https://www.goodfinance.org.uk/understanding-social-investment/types-social-investment) are treated more like equity.

Features such as the social outcomes that payment is contingent upon might be useful to users of the Data Specification. Modeling these unique features of specifically social investment forms of finance - e.g. of an SIB, a Community Share Issues - is important to consider within the conceptual framework. 

Are we able to include the payment-by-results element with the model as it is for other types of equity? Or does a Social Impact Bond require a separate definition?

**Comment by ScatteredInk** [comment]
We could potentially structure our measurements like this to account for future _targets_ as well as actual observations:

```
{
"id": "43000e63-1866-42a2-bdd1-6fe86c712232",
"type": "target",
"measurement": {....} 
}
``` 
For any conditional finance, we could then point to the ids of the target outcomes in an array. I think we would probably need an explanatory `notes` field to cover the complexity of some deals also.

For SIBs themselves, I think we need to see in detail how these are structured. From a quick scan, it seems like:

- the investor provides the capital upfront;
- the service provider therefore gets paid regardless;
- the commissioner may or may not pay up depending on results; 
- the risk therefore falls on the investor.

Is that also your reading of it? I'm not sure _how_ to categorise that, to be honest. It doesn't seem like equity or a traditional bond - it really is a uniquely 'social' investment.

**Comment by BobHarper1** [comment]
Also useful here https://www.bigsocietycapital.com/what-we-do/current-projects/public-service-reform/outcomes-contracts-social-impact-bonds

**Comment by BobHarper1** [comment]
@ScatteredInk  Yes, (sorry for the response delay) I have the same understanding. Having spoken to a few people about this, it's considered neither a loan nor credit, perhaps a type of quasi-equity or a completely unique instrument. 

People who have tried to fit this into current data publication alongside loans have struggled with the modelling. So I think, following [#20](https://github.com/SocialInvestmentDataLab/spec/issues/20#issuecomment-354984183) that it would be best treated as its own object, and I've added to the social impact bond branch:

```json
"SocialImpactBond" : {
            "type": "object",
            "title": "Social Impact Bond",
            "description": "A performance-based contract ('payment by results') which raises capital to pay for improved social outcomes and returns money to investors depending on outcome success.",
            "required": ["id"],
            "properties": {
                "id": {
                    "title": "Identifier",
                    "type": "string"
                },
                "currency": {
                    "title": "Currency",
                    "type": "string"
                },
                "capital": {
                    "title": "Initial Capital",
                    "description": "The amount of capital provided upfront to service providers by investors.",
                    "type": "number"
                },
                "maximumPayment": {
                    "title": "Maximum Return",
                    "description": "The total amount of return that can be repaid to investors covering all outcomes.",
                    "type": "number"
                },
                "deliveryAgency": {
                    "title": "Delivery Agency",
                    "description": "The contractor who ",
                    "$ref": "#/definitions/Organization"
                },
                "commissioner": {
                    "title": "Commissioner",
                    "description": "The organisation (e.g. government body) making payments based on measured outcomes.",
                    "$ref": "#/definitions/Organization"
                },
                "fundingProgramme": {
                    "title": "Funding Programme",
                    "description": "Funding programme that the finance forms a part of.",
                    "$ref": "#/definitions/FundingProgramme"
                }
            }
        }
```
